Cargo.lock

```diff
@@ -2417,22 +2433,30 @@ dependencies = [
 name = "itc-tls-websocket-server"
 version = "0.8.0"
 dependencies = [
+ "bit-vec",
+ "chrono 0.4.19",
  "env_logger",
+ "itp-sgx-crypto",
+ "itp-sgx-io",
  "log 0.4.17",
  "mio 0.6.21",
  "mio 0.6.23",
  "mio-extras 2.0.6 (git+https://github.com/integritee-network/mio-extras-sgx?rev=963234b)",
+ "rcgen",
  "rustls 0.19.0 (git+https://github.com/mesalock-linux/rustls?branch=mesalock_sgx)",
  "rustls 0.19.1",
  "sgx_tstd",
  "sgx_types",
+ "sp-core",
  "thiserror 1.0.31",
  "thiserror 1.0.9",
  "tungstenite 0.14.0 (git+https://github.com/integritee-network/tungstenite-rs-sgx?branch=sgx-experimental)",
  "tungstenite 0.15.0",
  "url 2.2.2",
  "webpki 0.21.4 (registry+https://github.com/rust-lang/crates.io-index)",
  "webpki 0.21.4 (git+https://github.com/mesalock-linux/webpki?branch=mesalock_sgx)",
+ "yasna 0.3.1",
+ "yasna 0.4.0",
```

Comment:
We have 2 versions of yasna? Probably one for sgx and the other in std? If possible, we should use the same versions for both features.

Reply:
It's not that easy to downgrade rcgen (duplicate lang item in crate `sgx_tstd with pem lib) After discussing it with @haerdib, we think it's better to update the sgx libraries (mesalock) rather than downgrade rcgen. I create a new issue: https://github.com/integritee-network/worker/issues/824